.github/workflows/release.yml

```diff
@@ -2,33 +2,27 @@ name: Release
 
 on:
   push:
-    branches:
```

Comment:
these push conditions are ORed, so the job runs on all pushes to master and all v* tags on any branch